HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealth.swift

```diff
@@ -668,7 +673,9 @@ extension GiniHealth: PaymentComponentsControllerProtocol {
         paymentDelegate?.didFetchedPaymentProviders()
     }
     
-    
+    public func didDismissPaymentComponents() {
```

Comment:
[Clarification]: Why did we name this method `didDismissPaymentComponents` and call inside `didDismissHealthSDK`, does it make sense to call it `didDismissSDK` as well